shelley/chain-and-ledger/formal-spec/address.tex

```diff
@@ -46,7 +53,16 @@ \section{Addresses}
 rewards from the proof of stake leader election. Appendix A
 of~\cite{delegation_design} explains this design choice.  The mechanism for
 transferring rewards from these accounts will be explained in
-Section~\ref{sec:utxo} and follows~\cite{chimeric}.
+Section~\ref{sec:utxo} and follows the approach outlined in the
+document~\cite{chimeric}.
+
+Note that, even though in the Cardano system, most of the
+accounting is UTxO-style, the reward addresses are a special case. They do not
+fall prey to many of the pitfalls of account-based accounting because of their restricted
```

Comment:
I'm not sure I like the idea of coming down so hard on this style without justification. Maybe just say that this style is only used here?